official/resnet/keras/keras_cifar_main.py

```diff
@@ -82,9 +82,7 @@ def parse_record_keras(raw_record, is_training, dtype):
   Returns:
     Tuple with processed image tensor and one-hot-encoded label tensor.
   """
-  image, label = cifar_main.parse_record(raw_record, is_training, dtype)
-  label = tf.compat.v1.sparse_to_dense(label, (cifar_main.NUM_CLASSES,), 1)
-  return image, label
+  return cifar_main.parse_record(raw_record, is_training, dtype)
```

Comment:
While I do a test, can you remove this method all together so it is not a pass through.

Reply:
Done, thank you!